conkit/core/contactmap.py

```diff
@@ -894,6 +918,72 @@ def remove_neighbors(self, min_distance=5, max_distance=sys.maxsize, inplace=Fal
                 contact_map.remove(tuple(contactid))
         return contact_map
 
+    def trim_region(self, region, inplace=False):
```

Comment:
this method is not required. see [`ContactMap.find`](https://github.com/rigdenlab/conkit/blob/cc74b643dc9524f1a229f2f10533eb894a7864c5/conkit/core/contactmap.py#L634)